skipper.yaml

```diff
@@ -47,3 +47,7 @@ env:
   CONNECTIVITY_CHECK_IMAGE: $CONNECTIVITY_CHECK_IMAGE
   HARDWARE_INFO_IMAGE: $HARDWARE_INFO_IMAGE
   INVENTORY_IMAGE: $INVENTORY_IMAGE
+  TARGET: $TARGET
+  OC_TOKEN: $OC_TOKEN
+  OC_SERVER: $OC_SERVER
+  SERVICE_NAME: $SERVICE_NAME
```

Comment:
can be changed?

Reply:
yes, backward compatibility - currently it's bm-inventory on psi
in the future will be assisted-service only and then we can remove this variable